spec/function-builder.spec.ts

```diff
@@ -132,7 +147,39 @@ describe('FunctionBuilder', () => {
       functions
         .region('asia-northeast1')
         .runWith({ timeoutSeconds: 600, memory: '256MB' });
-    }).to.throw(Error, 'TimeoutSeconds');
+    }).to.throw(Error, 'RuntimeOptions.timeoutSeconds');
+  });
+
+  it('should throw an error if user chooses a failurePolicy of type number', () => {
```

Comment:
I don't think it's necessary to test both string and number here as these tests should just be testing `failurePolicy` not being an object - otherwise it sends a message that string and number are somehow special.

Reply:
4059b00a73e8136c7dfbb51e234b0495d784c31d